static/css/custom.css

```diff
@@ -22,6 +22,12 @@ pre > code {
   transform: scale(1.1);
 }
 
+img {
+  max-width: 100%;
+  height: auto;
+  border-radius: 4px;
```

Comment:
Not sure if Karl will love this round corner style. I guess it won't be desirable to apply this style to inline images.